mitxpro/urls.py

```diff
@@ -80,6 +81,11 @@
     re_path(r"^receipt/(?P<pk>\d+)/", index, name="order-receipt"),
     re_path(r"^account-settings/", index, name="account-settings"),
     # Wagtail
+    re_path(
+        r"^images/([^/]*)/(\d*)/([^/]*)/[^/]*$",
```

Comment:
Does this URL pattern come from somewhere?

Reply:
https://docs.wagtail.io/en/v2.8/advanced_topics/images/image_serve_view.html#setup

It's awkward, I know. Seems like you shouldn't have to copy/paste a weird URL pattern to get this working, but that's what their doc suggests

Reply:
Now that I think about it, I should add a comment